src/pages/STG_SettingsManager.page

```diff
@@ -1,78 +1,74 @@
-<apex:page showHeader="true" sidebar="false" controller="STG_SettingsManager_CTRL" action="{!setTriggerSchedConfig}" docType="html-5.0" >
+<apex:page showHeader="true" sidebar="false" controller="STG_SettingsManager_CTRL" action="{!setTriggerSchedConfig}" docType="html-5.0" standardStylesheets="false">
```

Comment:
I don't like how the navbar is now the same background as the page.  It no longer looks like a separate section.  If you look at the navbar in lightning setup, that I assume you are trying to mimic, it does appear in its own section since they have a gray background on the current page they host.  I'd like us to at least add a vertical line demarcating the two.  Jade might have more input on whether some amount of difference in background should be used.

also, their highlight for an indented item still spans the whole width of the navbar.  Would be nice if you could do the same if that is relatively easy.

![image](https://cloud.githubusercontent.com/assets/908119/13760086/94035508-e9ee-11e5-95f6-45c022601386.png)